hazelcast/src/test/java/com/hazelcast/util/scheduler/SecondsBasedEntryTaskSchedulerTest.java

```diff
@@ -120,23 +106,25 @@ public void test_rescheduleEntry_postpone() {
     }
 
     @Test
+    // setting system property, not a ParallelTest !
+    @Category(QuickTest.class)
```

Comment:
Thanks @serkan-ozal. I had seen your before, but somehow I forgot it. You're right, I'll remove the comment.